test/nl/hannahsten/texifyidea/inspections/latex/LabelMissingInspectionTest.kt

```diff
@@ -0,0 +1,39 @@
+package nl.hannahsten.texifyidea.inspections.latex
+
+import com.intellij.testFramework.fixtures.BasePlatformTestCase
+import nl.hannahsten.texifyidea.testutils.writeCommand
+import org.junit.Test
+
+class LabelMissingInspectionTest : BasePlatformTestCase() {
+    override fun getTestDataPath(): String {
+        return "testData/inspections/latex"
+    }
+
+    override fun setUp() {
+        super.setUp()
+        myFixture.enableInspections(LatexMissingLabelInspection())
+    }
+
+    @Test
+    fun testMissingLabelWarnings() {
+        val testName = getTestName(false)
+        myFixture.configureByFile("$testName.tex")
+        myFixture.checkHighlighting(false, false, true, false)
+    }
+
+    @Test
+    fun testInsertCommandLabelQuickFix() {
+        val testName = getTestName(false)
+        myFixture.configureByFile("${testName}_before.tex")
+        do {
+            // we need to collect the fixes again after applying a fix because otherwise
+            // the problem descriptors use a cached element from before the applying the fix
+            val allQuickFixes = myFixture.getAllQuickFixes()
+            val fix = allQuickFixes.firstOrNull()
+            writeCommand(myFixture.project) {
+                fix?.invoke(myFixture.project, myFixture.editor, myFixture.file)
```

Comment:
I didn't know testing inspections was this easy. This will definitely help to create tests next time I'm fixing an inspection :)